qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/QuteDefinition.java

```diff
@@ -295,7 +295,7 @@ private CompletableFuture<List<? extends LocationLink>> findDefinitionFromParame
 		if (!isWhenSection(whenSection)) {
 			return NO_DEFINITION;
 		}
-		if (((CaseSection) caseSection).isCaseOperator(parameter)) {
+		if (((CaseSection) caseSection).isCaseOperator(parameter) && caseSection.getParameters().size() > 1) {

```

Comment:
what about?

`{#is in i|n}`

where `in` is an enum value.

Reply:
In that case isCaseOperator() will return false, and it should find the definition